deepchem/models/torch_models/layers.py

```diff
@@ -2,6 +2,7 @@
 try:
```

Comment:
Can you add unit tests for the new layers?

Reply:
Working on it :+1: